core/src/main/java/discord4j/core/object/entity/Message.java

```diff
@@ -702,6 +714,11 @@ public MessageEditMono edit() {
         return MessageEditMono.of(this);
     }
 
+    public Mono<Message> forward(TopLevelGuildMessageChannel channel) {
```

Comment:
a little unsure about this method.. mostly if is valid just allow pass the channel... or add a helper method in channel like createForwardMessage with a Message like argument for autofill the spec

Reply:
You can forward messages to DMs and private groups so just use a Snowflake as argument?

Reply:
Yeah.. i modify this method for allow just pass the MessageChannel with the createMessage method and also include a basic forward for just pass channel Id with the same behaviour for try to parse to the MessageChannel with the method to forward.